docs/_posts/Mary-Sci/2023-08-04-legclf_parameter_efficient_prediction_en.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: model
+title: Legal Parameter Efficiency Prediction in Domain-Specific Documents
+author: John Snow Labs
+name: legclf_parameter_efficient_prediction
```

Comment:
Please change the name to `legal_law_stack_exchange`. See what needs to be done on the S3.